bangazonapi/views/profile.py

```diff
@@ -178,14 +178,10 @@ def cart(self, request):
                 open_order = Order.objects.get(
                     customer=current_user, payment_type=None)
                 line_items = OrderProduct.objects.filter(order=open_order)
-                line_items = LineItemSerializer(
-                    line_items, many=True, context={'request': request})
 
                 cart = {}
                 cart["order"] = OrderSerializer(open_order, many=False,
                                                 context={'request': request}).data
-                cart["order"]["line_items"] = line_items.data
-                cart["order"]["size"] = len(line_items.data)
```

Comment:
We do want to see the `size` property that shows how many line items are on the order. Look at the `count()` method, and see how you can apply it here to accomplish this.
NOTE: You will _not_ need to replace any of the other lines of code you removed

Reply:
Got it!